<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addRequestBtn">
        <property name="text">
         <string>Добавить новый запрос</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="requestsTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>основное</string>
    </property>
    <addaction name="reloadAction"/>
    <addaction name="separator"/>
    <addaction name="leaveAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>вид</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>отображение</string>
     </property>
     <widget class="QMenu" name="menu_4">
      <property name="title">
       <string>работы</string>
      </property>
      <addaction name="displayWorksAsBlock"/>
      <addaction name="displayWorksAsStrings"/>
     </widget>
     <addaction name="menu_4"/>
    </widget>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Администрирование</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>запросы</string>
     </property>
     <addaction name="statusManager"/>
    </widget>
    <addaction name="menu_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="reloadAction">
   <property name="text">
    <string>обновить</string>
   </property>
   <property name="toolTip">
    <string>обновить</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="leaveAction">
   <property name="text">
    <string>выход</string>
   </property>
  </action>
  <action name="displayWorksAsBlock">
   <property name="text">
    <string>блоки</string>
   </property>
  </action>
  <action name="displayWorksAsStrings">
   <property name="text">
    <string>строки</string>
   </property>
  </action>
  <action name="statusManager">
   <property name="text">
    <string>Статусы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
